views/layouts/main.hbs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Articles and Products</title>
+</head>
+{{> header}}
```

Comment:
If you're header.hbs file is empty, you should delete that file and delete this line.